jujugui/static/gui/src/app/components/model-actions/_model-actions.scss

```diff
@@ -39,11 +36,24 @@
     width: 0;
   }
 
-  &--initial &__export {
-    display: none;
+  &--initial {
+    &__export,
+    &__share {
+      display: none;
+    }
   }
 
-  &--initial &__import {
+  .link {
```

Comment:
The short answer is yes: it's a standard pattern. That said, I'm not sure where the base style originates, but it applies the following styles to the elements:

~~~~
cursor: pointer;
color: #333;
text-decoration: underline;
~~~~